src/cborparser.c

```diff
@@ -915,6 +915,12 @@ static uintptr_t iterate_memcmp(char *s1, const uint8_t *s2, size_t len)
     return memcmp(s1, (const char *)s2, len) == 0;
 }
 
+static uintptr_t iterate_memcpy(char *dest, const uint8_t *src, size_t len)
```

Comment:
Why does this return true, and not the actual return value of memcpy?  This conversion of bool to int could very likely prevent at least 1 major compiler from eliding this wrapper.